src/android/com/outsystems/firebase/cloudmessaging/OSFirebaseCloudMessaging.kt

```diff
@@ -176,6 +191,19 @@ class OSFirebaseCloudMessaging : CordovaImplementation() {
         }
     }
 
+    private fun sendLocalNotificationWithPermission(args : JSONArray) {
```

Comment:
Should we be requesting this permission only when the `sendLocalNotification` method is called? What if we install the app in the device and only want to receive push notifications? Should they be requested in another moment? 

I was checking and it seems that for iOS we ask the permission when the app opens. We could consider doing the same.